Apps/WebClient/src/Server/Controllers/UserProfileController.cs

```diff
@@ -109,6 +116,12 @@ public async Task<IActionResult> CreateUserProfile(string hdid, [FromBody] Creat
         public IActionResult GetUserProfile(string hdid)
         {
             ClaimsPrincipal user = this.httpContextAccessor.HttpContext.User;
+            var jsonSettings = new JsonSerializerSettings()
+            {
+                Converters = new List<JsonConverter>() { new JsonClaimConverter(), new JsonClaimsPrincipalConverter(), new JsonClaimsIdentityConverter() },
```

Comment:
So why do we need all of this?

Reply:
unfortunately the claims serialization is not done by default (no default converters can do it). So these custom converters do the job.

Similar issues: https://github.com/codekoenig/AspNetCore.Identity.DocumentDb/issues/22